src/app/prevention/stop-the-breed/game-config.ts

```diff
@@ -0,0 +1,79 @@
+import { Item } from './models/Item';
+import { Element } from './models/Element';
+
+// For the background of the game
+
+export const backgroundConfig: Item = {
+  imageURL: '../../../assets/img/stop-the-breed/background_village.png',
+  positionX: 0,
+  positionY: 0,
+  width: 800,
+  height: 425,
+  messageText: '',
+  messageImageURL: '../../../assets/img/stop-the-breed/background2.png'
+};
+
+// For the elements that user can select in the game
+
+export const elementConfig: Array<Element> = [{
+  name: 'Bottle',
+  item: {
+    imageURL: '../../../assets/img/stop-the-breed/bottle.png',
+    positionX: 200,
+    positionY: 180,
+    width: 20,
+    height: 30,
+    messageText: 'They are potential breeding grounds for Mosquitoes. ' +
+      'Do look for them in your surroundings and remove them.',
+    messageImageURL: '../../../assets/img/stop-the-breed/message/bottle.jpg'
+  }
+}, {
+  name: 'Bowl',
+  item: {
+    imageURL: '../../../assets/img/stop-the-breed/bowl.png',
+    positionX: 30,
+    positionY: 150,
+    width: 30,
+    height: 30,
+    messageText: 'They are potential breeding grounds for Mosquitoes. ' +
+      'Do look for them in your surroundings and remove them.',
+    messageImageURL: '../../../assets/img/stop-the-breed/message/bowl.jpeg'
+  }
+}, {
+  name: 'Tyre',
+  item: {
+    imageURL: '../../../assets/img/stop-the-breed/tyre.png',
+    positionX: 695,
+    positionY: 325,
+    width: 60,
+    height: 90,
+    messageText: 'They are potential breeding grounds for Mosquitoes. ' +
+      'Do look for them in your surroundings and remove them.',
+    messageImageURL: '../../../assets/img/stop-the-breed/message/tyre.jpg'
```

Comment:
The relative paths for the URL may work fine but I think this would cause problems in future if the folder structure changes. Maybe we can split the url in such a way that making any changes to the folder structure will require no or minimal changes to the files. 
In this case the path should be changed at each and every place it is being referenced.

Reply:
Yes! Angular already provides the way for that. We had previously defined our 'assets' folder in the 'assets' property of Angular.json Didn't realise it earlier. Have pushed the required changes.